### PR TITLE
Feature/add treebeard

### PR DESCRIPTION
**Pull request by billyhunt**
Add the ability to add or remove an institution to either a single node, or a node and its children, depending on what the user would like.  

screen shot 2016-05-12 at 4 31 55 pm
screen shot 2016-05-12 at 4 31 50 pm

### CODE REVIEW

**Comment by brianjgeiger** [comment]
By and large, this looks good. We talked in person about only having, for any given transaction, a single dialog box at most. 

Also, and this will not be the most favorite thing you've ever read from me, I'd love some tests. Not the AJAX stuff, but the logic about adding and removing things, maybe if you can test about when to bring up which dialog box and what it should say, and so on. You'll probably have to pull some things from anonymous functions into named functions, and you might have to refactor your promises and callbacks a little, but it shouldn't be _too_ bad.

**Comment by brianjgeiger** [comment]
Pass done. 🐧

File: website/project/views/node.py [hunk 1]
```diff
@@ -976,6 +984,7 @@ def node_child_tree(user, node_ids):
             'category': node.category,
             'permissions': {
                 'view': can_read,
+                'is_admin': node.has_permission(user, 'read')
```
Comment (hmoco) [inline website/project/views/node.py:1003]: should that not say `admin`?

**Comment by billyhunt** [comment]
Moved to https://github.com/CenterForOpenScience/osf.io/pull/5726

**Pull request closed without merging**